CHANGELOG.md

```diff
@@ -4,6 +4,36 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
+## 2.0.0 - 2024-09-11
+
+### Added
+
+- ✨ Support Nextcloud 29.
+
+### Changed
+
+- 🗣️ Update translations.
+- 🔌 Upgrade dependencies.
+
+### Removed
+
+- 🧹 Drop support for Nextcloud 28.
+
+## 1.15.0 - 2024-01-22
+
+### Added
+
+- ✨ Support Nextcloud 28.
+
+### Changed
+
+- 🗣️ Update translations.
+- 🔌 Upgrade dependencies.
+
+### Removed
+
+- 🧹 Drop support for Nextcloud 27.
+
```

Comment:
Isn't it a breaking change?

Reply:
We were not really doing semantic versioning up until stable30 branch.

But you're right I confused the versioning and this is actually 1.16.0. (at least according to appinfo)